### ISSUE TITLE
Revamp Import Assertions tests

### ISSUE THREAD

**Issue by annevk** [web-platform-tests/wpt#42875]
https://github.com/web-platform-tests/wpt/pull/41156 decided to keep a lot of the coverage for the "assert" keyword. Unfortunately this might encourage implementers to go after the wrong thing. I think at best that coverage should be in `.tentative` but ideally we remove it and test that it's not supported.

@nicolo-ribaudo thoughts?

**Comment by nicolo-ribaudo** [comment]
I originally left those tests for browsers that only support `assert`, but given that Chrome now has a (flagged) `with` implementation and Safari/Firefox will only support `with`, they can probably be removed.

**Comment by domenic** [comment]
I don't think we should remove them, as they're still in the spec, and we need tests for even optional features to make sure they're not implemented incorrectly. But renaming to include `.optional` seems reasonable if there's a worry that people might implement these optional features, and I guess at least some parts of the spec community don't want that.

**Comment by annevk** [comment]
I didn't know about `.optional`. That sounds good! Hopefully in time TC39 gets to fully remove this.

**Comment by nicolo-ribaudo** [comment]
Feel free to assign this to me, I can find some time to do the rename next week
